src/RandomMapGenerator.py

```diff
@@ -223,24 +223,33 @@ def main() -> None:
     try:
         generate_num = int(args.generateNum)
     except ValueError:
-        print(f"error: Expected an integer for 'generateNum', but received a different type.")
+        print(
+            "error: Expected an integer for 'generateNum', "
+            "but received a different type."
+        )
         print(f"generateNum type: {type(args.generateNum)}")
 
     try:
         block_num = int(args.blockNum) if args.blockNum else 9
     except ValueError:
-        print(f"error: Expected an integer for option '--blockNum', but received a different type.")
+        print(
+            "error: Expected an integer for option '--blockNum', "
+            "but received a different type."
+        )
         print(f"blockNum type: {type(args.blockNum)}")
 
     try:
         item_num = int(args.itemNum) if args.itemNum else 10
     except ValueError:
-        print(f"error: Expected an integer for option '--itemNum', but received a different type.")
+        print(
+            "error: Expected an integer for option '--itemNum', "
+            "but received a different type."
+        )
         print(f"itemNum type: {type(args.generateNum)}")
 
     for i in range(generate_num):
-        entire_map, agent_position = make_map(block_num, item_num)
-        output_map("./generated_map/", f"RandMap_{i}", entire_map, 120, agent_position)
+        entire_map, agent_position = makeMap(block_num, item_num)
+        outputMap("./generated_map/", f"RandMap_{i}", entire_map, 120, agent_position)
```

Comment:
相対パス表記だと実行環境ごとに意図しない場所に出力される可能性があるので
`__file__`や`os.getcwd()`などから出力先を指定する方が良いかも

```suggestion
        outputMap(os.path.join(os.getcwd(),"generated_map", f"RandMap_{i}", entire_map, 120, agent_position))
```

Reply:
修正しました.
[コメントで指摘された点の修正](https://github.com/KPC-U16/MapGenerator/pull/18/commits/5e18006a2060fd81705ea83a0d3c37a6ea198be0)